src/arena.rs

```diff
@@ -249,9 +249,10 @@ impl<T> Arena<T> {
 
         // Hold the first value in a pointer to sidestep the borrow checker
```

Comment:
I suppose this comment is out of date now

Reply:
Yes, I've removed that line and force pushed.